copter/source/docs/loiter-mode.rst

```diff
@@ -49,6 +49,28 @@ Details for tuning :ref:`AltHold are on this wiki page <altholdmode_tuning>`.
 
 :ref:`VEL_XY_P <VEL_XY_P>` (shown as "Rate Loiter P, I and D") converts the desired speed towards the target to a desired acceleration.  The resulting desired acceleration becomes a lean angle which is then passed to the same angular controller used by :ref:`Stabilize mode <stabilize-mode>`.  **It is generally not required to adjust this.**
 
+
+.. _loiter-mode_tuning (from Copter version 3.6):
+.. note::
+
+Tuning the new Loiter mode (from Copter version 3.6)
+======
```

Comment:
need to be 
Tuning the new Loiter mode (from Copter version 3.6)
=========================================